docs/operate/control.mdx

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 4
+description: "Control running servers and clusters"
+---
+
+import Admonition from '@theme/Admonition';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Control servers/clusters
+
+This page describes how you can deploy a distributed Restate cluster.
+
+<Admonition type="note" title="Preview state">
+    The current distributed deployment state is a preview.
+    We reserve the right to make breaking changes.
+</Admonition>
+
+Starting with version 1.2, Restate includes a command line utility to connect to and control running servers called `restatectl`. This tool is specifically designed for system operators to manage the Restate servers, and is different from the application developer-focused [Restate CLI](/develop/local_dev#running-restate-server--cli-locally) which is used to manage service deployments and invocations.
+
+## Getting `restatectl`
+
+Currently `restatectl` is only distributed as part of the `restate-server` Docker image. As Restate 1.2 is not yet officially released, you can use the Docker image `ghcr.io/restatedev/restate:main` to access the latest version.
```

Comment:
Jack added automation to distribute `restatectl` in the same way as we distribute `restate`. Maybe we can already align the documentation that way?